scripts/datasets/otb2015.py

```diff
@@ -0,0 +1,69 @@
+"""this script is used to prepare Otb2015 dataset for tracking,
+which is Single Object Tracking benchmark"""
```

Comment:
Do we need another simple tutorial about "how to prepare otb dataset"? Just like the rest datasets?